src/java.base/share/classes/java/util/Locale.java

```diff
@@ -2074,7 +2072,7 @@ public String getDisplayVariant(Locale inLocale) {
      * Returns a name for the locale that is appropriate for display to the
      * user. This will be the values returned by getDisplayLanguage(),
      * getDisplayScript(), getDisplayCountry(), getDisplayVariant() and
-     * optional <a href="./Locale.html#def_locale_extension">Unicode extensions</a>
+     * optional {@linkplain Locale##def_locale_extension Unicode extensions}
```

Comment:
Can `Locale` be removed? It seems redundant. There are other locations with the same situation in this class.

Reply:
Yes, removed this and the other instances of `@linkplain` that use the _Locale_ prefix. I left the ones in `Locale.Builder` and `Locale.LanguageRange` since they are generated with different html pages and need the _Locale_ link.